portfolio/src/main/webapp/script.js

```diff
@@ -35,8 +35,10 @@ function addRandomFact() {
 /**
 * Retrieves comments when page loads.
 */
-const getComments = () => {
-  fetch('/comments').then(response => response.json()).then((comments) => {
+const getComments = () => {  
+  const numberOfComments = document.getElementById('numberOfComments').value;
+  const typeOfComments = document.getElementById('typeOfComments').value;
+  fetch(`/list-comments?numberOfComments=${numberOfComments}&orderBy=${typeOfComments}`).then(response => response.json()).then((comments) => {
```

Comment:
According to the style guide, Javascript has a column limit of 80 characters.

Reply:
I was not aware of that, thank you